### DISCUSSION TITLE
new bitwarden ios app - an error occured - login not possible with vaultwarden lxc

### DISCUSSION THREAD

**Discussion by domidetres** [tteck/Proxmox#3669] [General] [answered]
after updating to the new bitwarden ios app the access to vaultwarden lxc is not possible anymore.
please see: https://www.reddit.com/r/vaultwarden/comments/1f9beia/comment/llu4xs4/
Also bitwarden is saying contact vaultwarden support:
https://community.bitwarden.com/t/an-error-has-occured/72073/7

please help

**Comment by muppie** [comment, marked as answer]
I got it to work by updating vaultwarden to the latest version. It took forever but got there eventuelly. Now it seems to work.
